### DISCUSSION TITLE
FAQs for XIAO Series | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#807] [Q&A]
# FAQs for XIAO Series | Seeed Studio Wiki

XIAO-FAQ PAGE

https://wiki.seeedstudio.com/XIAO_FAQ/#comments

**Comment by johnpiker** [comment]
How can we access SD Card data directly through the Xiao ESP32S3 sense USB Port?
